WalletWasabi/Blockchain/Transactions/TransactionHistoryBuilder.cs

```diff
@@ -69,7 +53,7 @@ public List<TransactionSummary> BuildHistorySummary()
 						DateTime = dateTime,
 						Height = coin.Height,
 						Amount = coin.Amount,
-						Label = coin.Label,
+						Label = foundTransaction.Label,
```

Comment:
Let's use tx label instead of coin label. This will act like a traditional label and not like clusters like in the coin control.